errors_test.go

```diff
@@ -53,6 +53,31 @@ func TestWrap(t *testing.T) {
 	}
 }
 
+func TestWrapfNil(t *testing.T) {
```

Comment:
I'd you add these tests below the next block you won't have to adjust the line numbers below